src/plans/solver_state.jl

```diff
@@ -278,14 +361,65 @@ function update_storage!(a::AbstractStateAction, s::AbstractManoptSolverState)
     return a.keys
 end
 
+function _storage_key_true(nt::NamedTuple)
+    return map(key -> NamedTuple{(key,),Tuple{Bool}}(true), keys(nt))
```

Comment:
(Sorry for so many questions) – this reads for me as “for every key in nt we return _one_ NamedTuple” so we have an array of NamedTuples afterwards? This seems strange, but probably There is something I am missing here. I also have no clue why this is necessary since there is also nothing filtered or reduced?

And then nt is point or the vector NamedTuple?

Reply:
Questions are perfectly fine :slightly_smiling_face: . As I wrote above, I'm going to remove this function.